inlong-audit/audit-sdk/src/main/java/org/apache/inlong/audit/util/AuditManagerUtils.java

```diff
@@ -76,6 +86,89 @@ public static int buildAuditId(AuditIdEnum baseAuditId,
         return Integer.parseInt(auditPreFix + buildAuditIdSuffix(baseAuditId.getValue()), 2);
     }
 
+    public static AuditInformation buildAuditInformation(String auditType,
+            FlowType dataFlow,
```

Comment:
Plz don't use `dataFlow` here, dataFlow in InLong represents a ETL processing

Reply:
done